README.md

```diff
@@ -1,136 +1,583 @@
+    • <a href="#faq">FAQ</a>
+</p>
 
-⚠️ _Active work in progress! Subject to rapid major changes._ ⚠️
+## Overview
 
-A multi-file code editor component with live preview
+Playground Elements are a set of components for creating interactive code
+experiences on the web, with live updating previews. You can use Playground to:
 
-## Introduction
+- Embed interactive code examples in your documentation.
+- Build interactive tutorials and example galleries.
+- Build full-featured coding sandboxes (think Glitch or JSBin).
 
-`<playground-ide>` is a web component that allows you to embed multi-file, editable, live-preview code examples that use JavaScript, TypeScript, HTML and CSS. It's like a mini-IDE that you can embed many times in a page, and it works without a server!
+#### 🤯 No backend required
 
-## Features
+Unlike other coding environments, Playground never sends code to a backend
+server for compilation or serving. Instead, Playground uses a Service Worker to
+create a virtual URL-space that runs 100% within the browser. If you can host
+static files, you can host a Playground.
 
-### No server-required
+#### <img src="images/typescript.png" width="18" height="18"> TypeScript support
 
-`<playground-ide>` uses a service worker to send files from the main page to the preview iframe. Users can edit examples locally and the edited files are served to the iframe without ever hitting the network. This means you can use `<playground-ide>` with a static file server, and preview reloads are ultra-fast!
+Playground automatically compiles `.ts` files using TypeScript. Compilation
+happens in a Web Worker on a separate thread, so your page stays responsive.
 
-### Multi-file unbundled editing
+#### 🧩 Web Components
 
-`<playground-ide>` serves each file indivdually to the preview iframe, instead of bundling them first. This gives faster refresh times and means taht you can utilize the standard web platform features in your examples without a bundler getting in the way and potentially breaking things. The experience is very much like working wtih a static file server.
+Playground uses Web Components, so it doesn't require a framework. Yet it will
+play nicely with any framework you're already using, like React, Vue, and
+Angular.
 
-HTML can have multiple `<script>` and `<link>` tags, even dynamically added. CSS can use `@import` and `url()`. JavaScript can use `import.meta.url`, dynamic `import()`, and `fetch()`. It all just works.
+#### 🍱 Mix-and-match for custom layouts
 
-### Bare-module specifiers support
+Playground is broken up into small components like an editor, file picker, and
+preview. Mix-and-match components to create any layout you want, or just use
+`<playground-ide>` for an easy out-of-the-box experience.
 
-Standard JavaScript modules are great, but currently they lack one feature that has such overwhelming use and utility that we included support for it: base module specifiers.
+#### 🎨 Themable
```

Comment:
Do you think it's worth mentioning how you can define samples inline for quick authoring, or in files?

Also, do you want to open up the pre-renderer and mention that? Progressive enhancement is a big feature!